modules/sr/robot/compass.py

```diff
@@ -16,4 +16,4 @@ def get_heading(self) -> float:
         """
         x, _, z = self._compass.getValues()
         heading = atan2(x, z) % tau
-        return add_jitter(heading, 0, tau)
+        return add_independent_jitter(heading, 0, tau, 0.4, True)
```

Comment:
0.4 feels like a magic number here, where does that come from? If it's easy to calculate the stddev from a range, it could solve https://github.com/srobo/competition-simulator/pull/336#discussion_r615959663, too.